### PR TITLE
Added ek87 PCB support

### PR DESCRIPTION
**Pull request by eniigmakeyboards**
## Description
Added support for the ek87 PCB, designed for Eniigma Keyboards TKL products.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/eniigmakeyboards/ek87/config.h [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+/* disable these deprecated features by default */
+#ifndef LINK_TIME_OPTIMIZATION_ENABLE
+  #define NO_ACTION_MACRO
+  #define NO_ACTION_FUNCTION
+#endif
```
Comment (zvecr) [inline keyboards/eniigmakeyboards/ek87/config.h:198]:
```suggestion
#define NO_ACTION_MACRO
#define NO_ACTION_FUNCTION
```

File: keyboards/eniigmakeyboards/ek87/ek87.c [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    return process_record_user(keycode, record);
+}
+
+bool led_update_kb(led_t led_state) {
+    // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+    return led_update_user(led_state);
+}
+*/
```
Comment (zvecr) [inline keyboards/eniigmakeyboards/ek87/ek87.c:50]:
```suggestion

```

File: keyboards/eniigmakeyboards/ek87/keymaps/default_ansi/keymap.c [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/eniigmakeyboards/ek87/keymaps/default_ansi/keymap.c:1]: Keyboards **must** have a default keymap, which my suggestion is to rename the `default_ansi` folder to `default`

File: keyboards/eniigmakeyboards/ek87/rules.mk [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+# Otherwise, delete this section
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
```
Comment (zvecr) [inline keyboards/eniigmakeyboards/ek87/rules.mk:23]:
```suggestion

```

File: keyboards/eniigmakeyboards/ek87/rules.mk [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+VIA_ENABLE = yes
```
Comment (zvecr) [inline keyboards/eniigmakeyboards/ek87/rules.mk:43]: This needs to be in `keyboards/eniigmakeyboards/ek87/keymaps/via/rules.mk`

```suggestion
VIA_ENABLE = yes
```

File: keyboards/eniigmakeyboards/ek87/ek87-default_ansi.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{"keyboard": "ek87", "keymap": "default_ansi", "layout": "LAYOUT_ansi", "layers": [["KC_ESC", "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_F6", "KC_F7", "KC_F8", "KC_F9", "KC_F10", "KC_F11", "KC_F12", "KC_PSCR", "KC_SLCK", "KC_PAUS", "KC_GRV", "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0", "KC_MINS", "KC_EQL", "KC_BSPC", "KC_INS", "KC_HOME", "KC_PGUP", "KC_TAB", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_LBRC", "KC_RBRC", "KC_BSLS", "KC_DEL", "KC_END", "KC_PGDN", "KC_CAPS", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "KC_SCLN", "KC_QUOT", "KC_ENT", "KC_LSFT", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH", "KC_RSFT", "KC_UP", "KC_LCTL", "KC_LGUI", "KC_LALT", "KC_SPC", "KC_RALT", "KC_RGUI", "KC_APP", "KC_RCTL", "KC_LEFT", "KC_DOWN", "KC_RGHT"]], "author": "", "notes": "", "version": 1, "source_ip": "172.18.0.7", "documentation": "This file is a configurator export. You can compile it directly inside QMK using the command `bin/qmk compile ek87-default_ansi.json`"}
```
Comment (fauxpark) [inline keyboards/eniigmakeyboards/ek87/ek87-default_ansi.json:1]: Please delete this file.

File: keyboards/eniigmakeyboards/ek87/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,11 @@
+#include QMK_KEYBOARD_H
+
+/* THIS FILE WAS GENERATED!
+ *
+ * This file was generated by qmk-compile-json. You may or may not want to
+ * edit it directly.
+ */
+
```
Comment (fauxpark) [inline keyboards/eniigmakeyboards/ek87/keymaps/default/keymap.c:8]:
```suggestion
```

File: keyboards/eniigmakeyboards/ek87/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,14 @@
+#include QMK_KEYBOARD_H
+
+/* THIS FILE WAS GENERATED!
+ *
+ * This file was generated by qmk-compile-json. You may or may not want to
+ * edit it directly.
+ */
+
```
Comment (fauxpark) [inline keyboards/eniigmakeyboards/ek87/keymaps/via/keymap.c:8]:
```suggestion
```

File: keyboards/eniigmakeyboards/ek87/rules.mk [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+# Build Options
```
Comment (fauxpark) [inline keyboards/eniigmakeyboards/ek87/rules.mk:13]:
```suggestion

# Build Options
```

File: keyboards/eniigmakeyboards/ek87/rules.mk (hunk 5 above)
Comment (eniigmakeyboards) [reply keyboards/eniigmakeyboards/ek87/rules.mk:32]: Fixed

File: keyboards/eniigmakeyboards/ek87/keymaps/default_ansi/keymap.c (hunk 3 above)
Comment (eniigmakeyboards) [reply keyboards/eniigmakeyboards/ek87/keymaps/default_ansi/keymap.c]: Fixed

File: keyboards/eniigmakeyboards/ek87/ek87-default_ansi.json (hunk 6 above)
Comment (eniigmakeyboards) [reply keyboards/eniigmakeyboards/ek87/ek87-default_ansi.json]: Fixed

File: keyboards/eniigmakeyboards/ek87/rules.mk (hunk 5 above)
Comment (zvecr) [reply keyboards/eniigmakeyboards/ek87/rules.mk:33]: Sorry, my suggestion was slightly off, this also needs to be removed here too.

Comment (eniigmakeyboards) [reply keyboards/eniigmakeyboards/ek87/rules.mk]: Are you just talking about the `VIA_ENABLE = yes` line? If so, fixed

File: keyboards/eniigmakeyboards/ek87/ek87.h [hunk 10]
```diff
@@ -0,0 +1,203 @@
+/* Copyright 2020 adamws
```
Comment (noroadsleft) [inline keyboards/eniigmakeyboards/ek87/ek87.h:1]: ## General thought on this file

I'm having the sensation that the the Split Backspace and Split Shift layout macros are superfluous and could be deleted. You could probably go with just the `_all`, `_ansi`, `_ansi_7uspace`, `_iso` and `_iso_7uspace` layouts and that'd be enough for the overwhelming majority of users, and this doesn't prevent the available layouts from being extended later.

Also, IMO some of the layout macros should be renamed, to enable support for the [Community Layouts feature](https://docs.qmk.fm/#/feature_layouts).

- `LAYOUT_ansi` -> `LAYOUT_tkl_ansi`
- `LAYOUT_ansi_7uspace` -> `LAYOUT_tkl_ansi_tsangan`
- `LAYOUT_iso` -> `LAYOUT_tkl_iso`
- `LAYOUT_iso_7uspace` -> `LAYOUT_tkl_iso_tsangan`

These renames should be made throughout your keyboard's codebase.

File: keyboards/eniigmakeyboards/ek87/ek87.h [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+  { k30, XXX, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, XXX, XXX, XXX, XXX }, \
+  { XXX, k41, k42, k43, k44, k45, k46, k47, k48, k49, k4a, k4b, XXX, k4d, k4e, XXX, k4g, XXX }, \
+  { k50, k51, XXX, k53, XXX, XXX, k56, XXX, XXX, XXX, k5a, k5b, XXX, k5d, k5e, k5f, k5g, k5h }  \
+}
+
+#define LAYOUT_iso( \
+    k00,      k02, k03, k04, k05,      k07, k08, k09, k0a, k0b, k0c, k0d, k0e, k0f, k0g, k0h, \
+    k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d,      k1f, k1g, k1h, \
+    k20,      k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, k2e, k2f, k2g, k2h, \
+    k30,      k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d,                     \
```
Comment (noroadsleft) [inline keyboards/eniigmakeyboards/ek87/ek87.h:137]: Community Layouts support requires all keycode arguments passed in the same order, which we can do here if `k2e` moves to the home row. (Based on your website's renders, ANSI-US Backslash and ISO Enter are the same matrix position.)

```suggestion
    k20,      k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d,      k2f, k2g, k2h, \
    k30,      k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, k2e,                \
```

File: keyboards/eniigmakeyboards/ek87/ek87.h [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+  { k30, XXX, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, XXX, XXX, XXX, XXX }, \
+  { k40, k41, k42, k43, k44, k45, k46, k47, k48, k49, k4a, k4b, XXX, k4d, XXX, XXX, k4g, XXX }, \
+  { k50, k51, XXX, k53, XXX, XXX, k56, XXX, XXX, XXX, k5a, k5b, XXX, k5d, k5e, k5f, k5g, k5h }  \
+}
+
+#define LAYOUT_iso_7uspace( \
+    k00,      k02, k03, k04, k05,      k07, k08, k09, k0a, k0b, k0c, k0d, k0e, k0f, k0g, k0h, \
+    k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d,      k1f, k1g, k1h, \
+    k20,      k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, k2e, k2f, k2g, k2h, \
+    k30,      k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d,                     \
```
Comment (noroadsleft) [inline keyboards/eniigmakeyboards/ek87/ek87.h:154]:
```suggestion
    k20,      k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d,      k2f, k2g, k2h, \
    k30,      k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, k2e,                \
```

Same change should go on the two macros below if you choose to keep them.

File: keyboards/eniigmakeyboards/ek87/info.json [hunk 13]
```diff
@@ -0,0 +1,99 @@
+{
+    "keyboard_name": "ek87",
+    "url": "",
+    "maintainer": "adamws",
+    "width": 18,
+    "height": 6,
```
Comment (noroadsleft) [inline keyboards/eniigmakeyboards/ek87/info.json:6]:
```suggestion
    "width": 18.25,
    "height": 6.25,
```

File: keyboards/eniigmakeyboards/ek87/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,5 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT_ansi(KC_ESC, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_PSCR, KC_SLCK, KC_PAUS, KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, KC_INS, KC_HOME, KC_PGUP, KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL, KC_END, KC_PGDN, KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT, KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, KC_UP, KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_RALT, KC_RGUI, KC_APP, KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT)
```
Comment (noroadsleft) [inline keyboards/eniigmakeyboards/ek87/keymaps/default/keymap.c:4]: Makes this easier to read/modify, as users are likely to copy this file.

```suggestion
    [0] = LAYOUT_tkl_ansi(
        KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR, KC_SLCK, KC_PAUS,
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_INS,  KC_HOME, KC_PGUP,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL, KC_END,   KC_PGDN,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,          KC_UP,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_APP,  KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
    )
```

File: keyboards/eniigmakeyboards/ek87/keymaps/via/keymap.c [hunk 15]
```diff
@@ -0,0 +1,8 @@
...
+	[3] = LAYOUT_all(KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS)
```
Comment (noroadsleft) [inline keyboards/eniigmakeyboards/ek87/keymaps/via/keymap.c:7]:
```suggestion
    [0] = LAYOUT_all(
        KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,           KC_PSCR, KC_SLCK, KC_PAUS,
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_TRNS, KC_BSPC, KC_INS,  KC_HOME, KC_PGUP,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_DEL,  KC_END,  KC_PGDN,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT, KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_TRNS,                   KC_UP,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_APP,  KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
    ),
	[1] = LAYOUT_all(
        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
    ),
	[2] = LAYOUT_all(
        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
    ),
	[3] = LAYOUT_all(
        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS
    )
```

File: keyboards/eniigmakeyboards/ek87/readme.md [hunk 16]
```diff
@@ -0,0 +1,17 @@
+# Eniigma Keyboards ek87
+
+![ek87](https://i.imgur.com/lmZYr1P.jpg)
```
Comment (noroadsleft) [inline keyboards/eniigmakeyboards/ek87/readme.md:3]:
```suggestion
![ek87](https://i.imgur.com/lmZYr1Pl.jpg)
```

File: keyboards/eniigmakeyboards/ek87/ek87.h (hunk 10 above)
Comment (eniigmakeyboards) [reply keyboards/eniigmakeyboards/ek87/ek87.h:1]: Fixed this

**Comment by eniigmakeyboards** [comment]
All changes have been made

File: keyboards/eniigmakeyboards/ek87/rules.mk (hunk 4 above)
Comment (zvecr) [reply keyboards/eniigmakeyboards/ek87/rules.mk:23]: change still needs to be made

**Comment by eniigmakeyboards** [comment]
Should be fixed now, just added the commit

**Comment by eniigmakeyboards** [comment]
> .

fixed? was just formatting of the comments

File: keyboards/eniigmakeyboards/ek87/info.json [hunk 17]
```diff
@@ -0,0 +1,99 @@
...
+		    {"label":"k40","x":0,"y":4.25,"w":1.25},{"label":"k41","x":1.24,"y":4.25},{"label":"k42","x":2.25,"y":4.25},{"label":"k43","x":3.25,"y":4.25},{"label":"k44","x":4.25,"y":4.25},{"label":"k45","x":5.25,"y":4.25},{"label":"k46","x":6.25,"y":4.25},{"label":"k47","x":7.25,"y":4.25},{"label":"k48","x":8.25,"y":4.25},{"label":"k49","x":9.25,"y":4.25},{"label":"k4a","x":10.25,"y":4.25},{"label":"k4b","x":11.25,"y":4.25},{"label":"k4d","x":12.25,"y":4.25,"w":2.75},{"label":"k4g","x":16.25,"y":4.25},
+		    {"label":"k50","x":0,"y":5.25,"w":1.25},{"label":"k51","x":1.25,"y":5.25,"w":1.25},{"label":"k53","x":2.5,"y":5.25,"w":1.25},{"label":"k56","x":3.75,"y":5.25,"w":6.25},{"label":"k5a","x":10,"y":5.25,"w":1.25},{"label":"k5b","x":11.25,"y":5.25,"w":1.25},{"label":"k5d","x":12.5,"y":5.25,"w":1.25},{"label":"k5e","x":13.75,"y":5.25,"w":1.25},{"label":"5f","x":15.25,"y":5.25},{"label":"k5g","x":16.25,"y":5.25},{"label":"k5h","x":17.25,"y":5.25}]
+        },
+        "LAYOUT_iso_splitrshift": {
```
Comment (noroadsleft) [inline keyboards/eniigmakeyboards/ek87/info.json:89]:
```suggestion
        "LAYOUT_tkl_iso_split_rshift": {
```

**Comment by noroadsleft** [comment]
For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

Thanks for contributing!

**Comment by eniigmakeyboards** [comment]
Thank you very much!

**Pull request merged**